ext/redhat/pxp-agent.init

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# pxp-agent        Init script for running the PCP Execution Protocol (PXP) agent daemon
+#
+# chkconfig: - 98 02
+#
+# description: Agent for the PCP Execution Protocol (PXP), based on
+#              the Puppet Communications Protocol (PCP).
+# processname: pxp-agent
+# config: /etc/sysconfig/pxp-agent
+
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+PATH=/opt/puppetlabs/puppet/bin:/usr/bin:/sbin:/bin:/usr/sbin
+export PATH
+
+[ -f /etc/sysconfig/pxp-agent ] && . /etc/sysconfig/pxp-agent
+lockfile=/var/lock/subsys/pxp-agent
+piddir=/var/run/puppetlabs
+pidfile=${piddir}/pxp-agent.pid
+pxp_agentd=/opt/puppetlabs/puppet/bin/pxp-agent
+pid=$(cat $pidfile 2> /dev/null)
+RETVAL=0
+
```

Comment:
Any reason we aren't checkng to see if pxp_agentd is executable here but we did in the debian? Same question for sles.